docs/standalone-kubelet-crio.md

```diff
@@ -0,0 +1,139 @@
+Kubelet standalone
+==================
+
+Kubelet may be installed headless, which is without the remaining kubernetes
+cluster components. In that "dummy" mode, kubelet only manages static pods
+locally and cannot schedule a pod onto other nodes.
+
+Example installation
+--------------------
+
+Given a Fedora 28 node already prepared and having a static inventory file for
+it, like:
+```
+$ cat ../hosts
+node ansible_host=<IP for ssh access> ansible_user=fedora
+[k8s-cluster]
+node
+```
+
+The following steps may be done to install headless kubelet with CRI-O and runc
+runtime. No networking plugins will be used, so pods only may be started in the
+host networking mode. Execute these example commands from the clonned kubespray
+repo directory:
+
+* Optional step to configure custom repos
+  (the required packages may be found in the stock updates repository as well):
+```
+$ cat > repos.yml << EOF
+- hosts: node
+  become: yes
+  gather_facts: yes
+  tasks:
+    - yum_repository:
+        name: paas7-openshift-origin311-candidate
+        description: 'OpenShift Origin Repo'
+        baseurl: https://cbs.centos.org/repos/paas7-openshift-origin311-candidate/x86_64/os/
+        enabled: yes
+        gpgcheck: no
+    - yum_repository:
```

Comment:
cri-o role does not have virt7-container-common-candidate. Which package is it required?

Reply:
One can install kubernetes-node 1.10.1 (0.fc28) from fedora updates repo, virt7-container-common-candidate repo just provides another cadence perhaps, tho it has only 1.10.0 currently. Anyway, that's a fully optional step and the repo.

Reply:
Ok, it's for 

>  pre_tasks:
>    - package: name=kubernetes-node state=present

kubespray use containerd or downloaded kubelet in install_docker.yml and install_host.yml.
If you can reuse these script, it seems better. In the future, it will support not only RHEL/Fedora.

But currently, cri-o is supported on only RHEL/Fedora. You can keep this code if you prefer.

Reply:
Yes, please let's keep this that way for now. And we can adjust things for future cases, once cri-o becomes wanted/installable for other OS types as well.